File: Algorithm/Programmers/Lv2/숫자의 표현.java
class Solution {
    public int solution(int n) {
        int cnt = 0;
        int sum = 0;
        
        for(int i = 1; i<=n; i++){
            sum = 0;
            for(int j = i; j<=n; j++){
                sum += i;
                if(sum == n) {
                    cnt++;
                    j = n+1;
                }
            }
        }
        return cnt;
    }
}

File: Algorithm/Programmers/Lv2/숫자의 표현.js
function solution(n) {
    let cnt =0;
    let sum = 0;
    for(let i =1; i<n; i++){
        sum +=i;
        for(let j = i+1; j<n; j++){
            sum+=j;
            if(sum===n){
                sum = 0;
                cnt++;
                break;
            }else if(sum>n){
                sum = 0; 
                break;
            }   
        }
    }
    return cnt+1;
}
